src/libraries/System.Private.CoreLib/src/System/TimeZoneInfo.Win32.cs

```diff
@@ -114,6 +114,16 @@ private static void PopulateAllSystemTimeZones(CachedData cachedData)
         {
             Debug.Assert(Monitor.IsEntered(cachedData));
 
+            cachedData._systemTimeZones ??= new Dictionary<string, TimeZoneInfo>(StringComparer.OrdinalIgnoreCase)
+            {
+                { UtcId, s_utcTimeZone }
+            };
+
```

Comment:
move inside Invariant block?

Reply:
no, this is good. It will happen only once, when it's null